### PR TITLE
Update plugin for linting and coveralls

### PR DESCRIPTION
**Pull request by rachelnicole**
Update package.json / eslint / coveralls.

Resolves #3 
Resolves #2 
Resolves #1

### CODE REVIEW

**Comment by Snugug** [comment]
Tests are failing. Looks like I missed https://github.com/punchcard-cms/input-plugin-range/blob/master/tests/plugin.js#L2. Please update

**Comment by Snugug** [comment]
Please update validation to make sure that the number put in falls within the min/max settings defined.

File: lib/validation.js [hunk 1]
```diff
@@ -22,5 +24,9 @@ module.exports = function rangeValidation(input, settings) {
     return `${input.target.name} cannot be left blank!`;
   }
 
+  if (input.target.value < settings.min || input.target.value > settings.max) {
+    return `${input.target.name} must be a value within given range!`;
```
Comment (scottnath) [inline lib/validation.js:28]: @rachelnicole same with this one. Please write a test which checks for a variable outside the current range.
Code coverage test:
https://travis-ci.org/punchcard-cms/input-plugin-range/builds/132905940
you can see where it shows line 28 is not covered:

```
-------------------------|----------|----------|----------|----------|----------------|
File                     |  % Stmts | % Branch |  % Funcs |  % Lines |Uncovered Lines |
-------------------------|----------|----------|----------|----------|----------------|
 input-plugin-range/     |      100 |      100 |      100 |      100 |                |
  index.js               |      100 |      100 |      100 |      100 |                |
 input-plugin-range/lib/ |    83.33 |     87.5 |      100 |    83.33 |                |
  validation.js          |    83.33 |     87.5 |      100 |    83.33 |             28 |
-------------------------|----------|----------|----------|----------|----------------|
All files                |     87.5 |     87.5 |      100 |     87.5 |                |
-------------------------|----------|----------|----------|----------|----------------|
```

**Comment by rachelnicole** [comment]
@scottnath full coverage plz recheck!

File: tests/validation.js [hunk 2]
```diff
@@ -17,21 +21,27 @@ const settings = {
...
-      empty: false,
+      min: 1,
+      max: 3,
     },
   },
 };
 
 
-// Valid input
+// // Valid input
```
Comment (scottnath) [inline tests/validation.js]: nitpik: single set of dual slashes on these lines 31/36 please

**Comment by scottnath** [comment]
@rachelnicole this is excellent and ready. Small nitpik comment on comments, then I'll merge!

**Comment by rachelnicole** [comment]
@scottnath fixed! good catch. oversight on my part 🐛

**Pull request merged**